app/bundles/get-started.js

```diff
@@ -0,0 +1,7 @@
+import CopyToClipboard from '../components/CopyToClipboard';
```

Comment:
Does it make sense to make this part of the base bundle to avoid the duplication?